validator.py

```diff
@@ -626,6 +630,83 @@ def does_key_have_value(self, key, value, yaml):
         return data == value
 
+
+    def scenes_with_transitions(self):
+        '''
+        Looks through the yaml file to make sure that every scene from 0 to n-1 has 
+        a transitions field
+        '''
+        data = copy.deepcopy(self.loaded_yaml)
+        scenes = data['scenes']
+        for i in range(0, len(scenes)-1):
+            if 'transitions' not in scenes[i]:
+                self.logger.log(LogLevel.WARN, "Key 'transitions'  must be provided within each entry in 'scenes' but is missing at scenes[" + str(i) + "]")
+                self.missing_keys += 1
+
+
+    def validate_action_params(self):
+        '''
+        Ensure that action parameters have valid values
+        '''
+        data = copy.deepcopy(self.loaded_yaml)
+        api = copy.deepcopy(self.api_yaml)
+        allowed_supplies = api['components']['schemas']['SupplyTypeEnum']['enum']
+        allowed_locations = api['components']['schemas']['InjuryLocationEnum']['enum']
+        allowed_categories = api['components']['schemas']['CharacterTagEnum']['enum']
+
+        scenes = data['scenes']
+        i = 0
+        for scene in scenes:
+            if 'action_mapping' in scene:
+                map = scene['action_mapping']
+                j = 0
+                for action in map:
+                    if 'parameters' in action:
+                        params = action['parameters']
+                        if 'treatment' in params:
+                            if params['treatment'] not in allowed_supplies:
+                                self.logger.log(LogLevel.WARN, "Key 'scenes[" + str(i) + "].action_mapping[" + str(j) + "].parameters.treatment' must be one of the following values: " + str(allowed_supplies) + " but is '" + params['treatment'] + "' instead.")
+                                self.invalid_values += 1                        
+                        if 'location' in params:
+                            if params['location'] not in allowed_locations:
+                                self.logger.log(LogLevel.WARN, "Key 'scenes[" + str(i) + "].action_mapping[" + str(j) + "].parameters.location' must be one of the following values: " + str(allowed_locations) + " but is '" + params['location'] + "' instead.")
+                                self.invalid_values += 1 
+                        if 'category' in params:
+                            if params['category'] not in allowed_categories:
+                                self.logger.log(LogLevel.WARN, "Key 'scenes[" + str(i) + "].action_mapping[" + str(j) + "].parameters.category' must be one of the following values: " + str(allowed_categories) + " but is '" + params['category'] + "' instead.")
+                                self.invalid_values += 1 
+                    j += 1
+            i += 1
+
+
+    def validate_mission_importance(self):
+        '''
+        Verifies that all characters with their mission importance appear
+        in the critical_ids list.
```

Comment:
Whatever is decided by Brian and Jennifer is fine, just let me know and I'll update whatever needs to be changed!

Reply:
They want it to be optional with defaults.  So the actions are:
- Add the default to `MissionImportanceEnum`
- Update the code to assume a missing value means 'normal'
- Edit `test.yaml` to add an entry in the `critical_ids` for `"Civilian_01": "none"` and we can remove `"mission_importance: none"` from Mike and Marine_burns_01, since these are now the defaults.

Reply:
Sorry you had to repeat that, I hadn't refreshed my page and the update on the comment was not showing up.

Reply:
Done!